transform.ml

```diff
@@ -8,11 +8,12 @@ let remove_empty_jmp prog =
       match (prog.(pc), prog.(pc')) with
       | (Goto l1, Label l2) when l1 = l2 && List.length pred.(pc') = 1 ->
           remove_empty_jmp (pc+2)
+      | (Label _, _) when pred.(pc') = [pc-1] ->
+          remove_empty_jmp pc'
```

Comment:
Looking at this code against I'm mildly confused as to why it is `pred.(pc')` and `pc-1` -- if I understand correctly, `pc-1` is `pc'-2`.